client/dashboard/data/user-tax-details.ts

```diff
@@ -0,0 +1,92 @@
+import wpcom from 'calypso/lib/wp';
```

Comment:
Can we rename the filename prefix from `user-` to `me-`, for consistency with existing files?

Reply:
Renamed in e5e42b2.